sample-functions/apikey-secret/README.md

```diff
@@ -2,7 +2,7 @@
 
 This function returns access denied, or unlocked depending on whether your header for X-Api-Key matches a secret in the cluster called `secret_api_key`.
 
-See the [secure secret management guide](../guide/secure_secret_management.md) for more information on secrets.
+See the [secure secret management guide](../../guide/secure_secret_management.md) for more information on secrets.
```

Comment:
Is this correct? I changed it recently so didn’t expect it to change again.

Reply:
Path is `/master/sample-functions/apikey-secret/README.md`

`../` takes you into `sample-functions`.

guide is at `/master/guide/`

Try it:

Current - https://github.com/openfaas/faas/tree/master/sample-functions/apikey-secret

Fixed - https://github.com/rgee0/faas/tree/fixSecretSample/sample-functions/apikey-secret

(I suspect you fixed it in `ApiKeyProtected-Secrets`)